pages/nfts/[address].tsx

```diff
@@ -151,7 +176,10 @@ export default function NftByAddress({
   const hasDefaultListing = Boolean(defaultListing);
   const offer = nft?.offers.find((offer) => offer.buyer === publicKey?.toBase58());
   const hasAddedOffer = Boolean(offer);
-  const offers = nft?.offers || [];
+  const offers = (nft?.offers || []);
+  const sortedOffers = Array.from(offers).sort((offerA ,offerB) => { 
+    return offerA.createdAt < offerB.createdAt ? 1 : -1
+  });
```

Comment:
TODO: Move sort to query.

In this state its a good candidate for useMemo so the offers aren't re-sorted on every render.

Reply:
```

  const { offers, sortedOffers, topOffers } = useMemo(() => {
    const offers = nft?.offers || [];
    const sortedOffers = Array.from(offers).sort((offerA, offerB) => {
      return offerA.createdAt < offerB.createdAt ? 1 : -1;
    });
    const topOffers = offers?.slice()?.sort((a, b) => Number(b?.price) - Number(a?.price)) || [];
    return {
      offers,
      sortedOffers,
      topOffers,
    };
  }, [nft?.offers]);
  ```

added this to dev